python_modules/libraries/dagster-dbt/dagster_dbt_tests/dbt_projects/test_dagster_dbt_unit_tests/models/staging/stg_payments.sql

```diff
@@ -1,3 +1,9 @@
+{{
+    config(
+        tags=["test"],
+    )
+}}
```

Comment:
maybe apply the tags to all models in `dbt_project.yml`? I noticed we didn't apply the tag to `customers.sql`, which is the model that has the unit test. I would want to see the interaction with a selection on a model with a unit test

Reply:
Done in [1d3daf4](https://github.com/dagster-io/dagster/pull/23232/commits/1d3daf41f8a1b2171a10a7088cea81101dc346f4)